### PR TITLE
Add installation to website, fix  styling, icon

### PR DESCRIPTION
**Pull request by LuisaE**
Add an installation page to the website (documentation). And fix the extension icon not showing up on the dev page. 

Captura de Tela 2023-01-29 às 2 54 46 PM

step4

### CODE REVIEW

**Comment by sukritsangvong** [comment]
Can u change the chrome extension pick to English? lol Here is a screenshot from mine 
image

**Comment by LuisaE** [comment]
> 

Thanks! @sukritsangvong Can you pull the branch, clean the errors, and take a new screenshot, please? So it shows up with the logo, correct version, and no errors haha

**Comment by sukritsangvong** [comment]
o. I thought the w in the icon section is the wikipedia's W all this time lol.

File: wikichange/src/manifest.json [hunk 1]
```diff
@@ -3,17 +3,18 @@
...
     "minimum_chrome_version": "88",
     "icons": {
-        "16": "assets/icons/wikiIcon16.png"
+        "16": "assets/icons/icon16.png",
+        "48": "assets/icons/icon16.png"
     },
     "action": {
         "default_icon": {
-            "16": "assets/icons/wikiIcon16.png"
+            "16": "assets/icons/icon_16.png"
```
Comment (sukritsangvong) [inline wikichange/src/manifest.json:17]: This doesn't match the name of the icon file

**Review by sukritsangvong — CHANGES_REQUESTED** [review]

**Comment by sukritsangvong** [comment]
> > 
> 
> Thanks! @sukritsangvong Can you pull the branch, clean the errors, and take a new screenshot, please? So it shows up with the logo, correct version, and no errors haha

image

**Comment by sukritsangvong** [comment]
> > > 
> > 
> > 
> > Thanks! @sukritsangvong Can you pull the branch, clean the errors, and take a new screenshot, please? So it shows up with the logo, correct version, and no errors haha
> 
> image

image

File: wikichange/src/manifest.json (hunk 1 above)
Comment (LuisaE) [reply wikichange/src/manifest.json:17]: Typo, thanks

Comment (sukritsangvong) [inline wikichange/src/manifest.json]: There are three places where you are using the icon.png. Need to match them all.

**Review by sukritsangvong — CHANGES_REQUESTED** [review]

**Review by sukritsangvong — APPROVED** [review]

**Pull request merged**